Symfony/CS/Tests/Fixer/Symfony/TrimArraySpacesFixerTest.php

```diff
@@ -0,0 +1,355 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Jared Henderson <jared@netrivet.com>
+ */
+class TrimArraySpacesFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+
+            array(
+                "<?php \$foo = array('foo');",
+                "<?php \$foo = array( 'foo' );",
+            ),
+
+            array(
+                "<?php \$foo = ['foo'];",
+                "<?php \$foo = [ 'foo' ];",
```

Comment:
the test using the short array syntax will not work on PHP 5.3 because it will throw a parse error when tokenizing the code. this is why it should be skipped